bankdroid-legacy/src/main/java/eu/nullbyte/android/urllib/Urllib.java

```diff
@@ -226,6 +227,24 @@ public HttpResponse openAsHttpResponse(String url, HttpEntity entity, HttpMethod
             request.addHeader(headerKeys[i], headerVals[i]);
         }
 
+        HttpRequestRetryHandler retryHandler = new HttpRequestRetryHandler() {
+
+            public boolean retryRequest(IOException exception, int executionCount,
+                                        HttpContext context) {
+                // retry a max of 5 times
+                if(executionCount >= 5){
```

Comment:
What do you think about moving the integer into a constant e.g. `MAX_RETRY_ATTEMPTS`?